web/ui/react-app/src/pages/graph/ExpressionInput.tsx

```diff
@@ -215,12 +215,10 @@ class ExpressionInput extends Component<ExpressionInputProps, ExpressionInputSta
                   value={value}
                 />
                 <InputGroupAddon addonType="append">
-                  <Button className="btn-light border" title="Open metrics explorer" onClick={this.openMetricsExplorer}>
+                  <Button className="metrics-explorer" title="Open metrics explorer" onClick={this.openMetricsExplorer}>
```

Comment:
Nice, I like how it looks now! The class name is a bit confusing since `metrics-explorer` is also used for the metrics explorer itself. Maybe `metrics-explorer-btn` or something like that would be clearer?

Reply:
I was hoping that being able to target it with `button.foo` is enough to make it clear but let's rename it for clarity

Reply:
Yep, I just think the class name itself would have been a bit of a misnomer otherwise... thanks!